docs/govuk-verify.md

```diff
@@ -33,6 +33,23 @@ You can check that the VSP is running ok by hitting the healthcheck URL:
 curl localhost:50300/admin/healthcheck
 ```
 
+### How to complete the Verify user journey in local development
+
+After beginning the Verify flow, you will be redirected to a URL which looks
+something like
+https://compliance-tool-reference.ida.digital.cabinet-office.gov.uk/SAML2/SSO,
```

Comment:
Visiting this URL returns a `405` error. Maybe we need to include the actual JSON instead?

Reply:
I wasn't intending that anybody follow this link; I just wanted to give an example of what the URL looks like, so that they know which part of the flow I'm talking about.

Reply:
Ah, OK, I wonder if there is a way we can mark this so that it looks less like a visitable link and isn't interpreted as such in Markdown. Maybe wrap it in quote marks? Or backticks?

Reply:
Good idea - I've added backticks.